### ISSUE TITLE
Non-functioning Device Removal

### ISSUE THREAD

**Issue by WardsParadox** [micromdm/micromdm#394]
If a device is permanently damaged and can not be removed via un-enrollment via `mdmctl apply block` method, there is no way to remove that device from the MDM. A command to nuke device from the database.

**Comment by jessepeterson** [comment]
For devices that have been added by DEP (serial number only) do we plan to support removing those as well (and not just via UDID as #444 solved)? Can close this issue if the answer is no. :)

**Comment by knightsc** [comment]
I could probably add support to the remove command to do it by serial number as well without to much trouble. I remember discussing the fact that the main spec says UDID is required and Serial is optional but DEP is the other way around right? Only a serial and no UDID?

**Comment by jessepeterson** [comment]
Right. UDID is always required for an *enrolled/enrolling* device. However, when DEP syncs you only get a serial number from the DEP API. I.e. this is for the time between when a new device is added to the DEP MDM server and the device actually attempts enrollment to the MDM server.
